src/containers/Proposals/index.tsx

```diff
@@ -1,10 +1,72 @@
-import { Box, Typography } from '@mui/material'
+import React from 'react'
+import { Box, InputAdornment, Typography, Chip, Button } from '@mui/material'
+import Card from 'components/Card/Card'
+import SearchIcon from '../../assets/vectors/search-icon.svg'
+
+import { InputContainer, styles } from './styles'
+import Proposal from './Proposal'
 
 const Proposals = () => {
   return (
-    <Box>
-      <Typography>Proposals</Typography>
-    </Box>
+    <>
+      <Box style={styles.stickyHeader}>
+        <Typography style={styles.headerStyle}>Proposals</Typography>
+        <Typography
+          style={styles.subheaderStyle}
+          variant="subtitle1"
+          color="text.secondary"
+        >
+          Here you can see the existing proposals’ statuses or create new one
```

Comment:
I'd suggest changing it to: "Here you can create new Proposal or check existing proposals status"